lms/static/sass/views/_account-settings.scss

```diff
@@ -36,45 +39,269 @@
+                        color: #7a7979;
+                        width: 100%;
+                        height: 48px;
+                        box-shadow: none;
+                    }
+
+                    .u-field-link {
+                        @extend %ui-clear-button;
 
-    a span {
-        &:hover, &:focus {
-            color: $pink;
-            text-decoration: none !important;
+                        // set styles
+                        @extend %btn-pl-default-base;
+                        @include font-size(18);
+                        width: 100%;
+                        border: 1px solid #0078be;
+                        color: #0078be;
+                        padding: 11px 14px 11px 14px;
+                        line-height: normal;
+                    }
+
+                    #u-field-value-username {
+                        padding-top: ($baseline/2);
+                    }
+                }
+
+                .social-field-linked {
+                    background: #fcfcfc;
+                    box-shadow: 0 1px 2px 1px rgba(167, 164, 164, 0.25);
+                    padding: 1.25rem 1.25rem;
+                    box-sizing: border-box;
+                    margin: 10px;
+                    width: 100%;
+
+                    .field-label {
+                        font-size: 24px;
+                    }
+
+                    .u-field-social-help {
+                        display: inline-block;
+                        padding: 20px 0 6px 0;
+                    }
+
+                    .u-field-link {
+                        border: none;
+                        margin-top: $baseline;
+                        @include font-size(14);
+                        font-weight: 600;
+                        text-align: left;
+                        padding: 0;
+
+                        &:focus {
+                            background-color: transparent;
+                            color: #126f9a;
+                            border: none;
+                        }
+
+                        &:hover {
+                            background-color: transparent;
+                            color: #126f9a;
+                            border: none;
+                        }
+                    }
+                }
+
+                .social-field-unlinked {
+                    background: #f2f1f1;
+                    box-shadow: 0 1px 2px 1px rgba(167, 164, 164, 0.25);
+                    padding: 1.25rem 1.25rem;
+                    box-sizing: border-box;
+                    text-align: center;
+                    margin: 10px;
+                    width: 100%;
+
+                    .field-label {
+                        font-size: 24px;
+                        text-align: center;
+                    }
+
+                    .u-field-link {
+                        margin-top: $baseline;
+                        @include font-size(14);
+                        font-weight: 600;
```

Comment:
Please use the already defined font-weight variables https://github.com/edx/edx-platform/blob/76b8e2e89761c30ac7aa2fa5e8aca4dc35d9002b/lms/static/sass/base/_variables.scss#L320